src/main/java/Problem1/ValidParentheses.java

```diff
@@ -1,11 +1,56 @@
 package Problem1;
 
+import Problem2.ListNode;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class ValidParentheses {
 
     // Do not change signature (function name, parameters, return type)
     public static boolean isValid(String str) {
         // homework
         // !!! must use ArrayStack or LinkedListStack from problem 1
-        return false; // place holder
+        Stack<Character> list = new LinkedListStack<>();
+        Stack<Character> list2 = new LinkedListStack<>();
```

Comment:
this idea of this algorithm is described in class. it should use one stack. I don't understand why there's 2 stacks here.